exasol/ds/sandbox/lib/dss_docker/create_image.py

```diff
@@ -50,28 +51,26 @@ def _ansible_config(self) -> ConfigObject:
             slc_version=SLC_VERSION,
         )
 
+    def _docker_file(self) -> Path:
+        return (
+            importlib_resources
```

Comment:
This is not always a path it is a traversable. Usually, it is the best to read the content of the traversable which could point into a archive file. The docker python SDK for build supports fileobj, it is probably best to get a fileobj from the traversable and use this